admin_guide/backup_restore.adoc

```diff
@@ -200,6 +200,77 @@ that {product-title} was previously installed.
 $ oc create -f mycluster.template.yaml
 ----
 
+[[backup-restore-recovering-etcd-hosts]]
+== Recovering etcd Hosts
+
+In cases where etcd hosts have failed, but you have at least one host still running, you can use that one host to recover etcd hosts without downtime.
+
+[NOTE]
+====
+The following presumes you have backups of the */etc/etcd* configuration for the etcd hosts.
+====
+
+. Add the desired number of hosts to the cluster.
+
+. Upgrade etcd in the surviving node with the provided version:
++
+----
+# rpm -Fvh etcd-2.3.7-3.el7.bz1350875.1.x86_64.rpm
```

Comment:
Why are we not using yum here, or asking folks to check the version with yum/rpm? 

If I am not mistaken we need this version to successfully do backups (with out error). So do we need a NOTE?